### PR TITLE
Implementation of movie list

### PR DESCRIPTION
**Pull request by evvis**
Movie_list component creating. 
Movie service for API usage, encapsulating request. 
Output of the received data to the cards. Layout.

### CODE REVIEW

File: src/app/movie-list/movie-list.component.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<div class="row">
+  <div *ngFor="let movie of movie_list | async">
```
Comment (maximprokopchuk) [inline src/app/movie-list/movie-list.component.html]: You should move this markup to separate component (you can name it, for example, `movie-card`)

File: src/app/movie-list/movie-list.component.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      <div class="col s4 m2">
```
Comment (maximprokopchuk) [inline src/app/movie-list/movie-list.component.html]: This markup is broken for different screen widths 
image 
image

File: src/app/services/movie.ts [hunk 3]
```diff
@@ -0,0 +1,43 @@
+export class Movie {
```
Comment (maximprokopchuk) [inline src/app/services/movie.ts]: You use this class as interface (you don't create Movies instances `new Movie(...)`), it is needed just for typings, so you should replace this definition to interface definition

**Review by maximprokopchuk — CHANGES_REQUESTED** [review]

**Comment by evvis** [comment]
Renamed the "movie-list" component in "movieList"
A "movieCard" component has been created, the contents of "movieList" have been transferred to it.
Fixed layout.
The movie class is translated into the interface.

File: src/app/movieList/movieCard/movieCard.component.html [hunk 4]
```diff
@@ -0,0 +1,29 @@
+<div class="row">
```
Comment (maximprokopchuk) [inline src/app/movieList/movieCard/movieCard.component.html]: not  sure we need for rows for each card, bat leave it as it is now. will refactor later

File: src/app/movieList/movieCard/movieCard.component.css [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+.row:after {
+  content: normal;
+}
+
+.card {
+  height: 27.875em;
```
Comment (maximprokopchuk) [inline src/app/movieList/movieCard/movieCard.component.css]: I said you should delete fixed height, it brokes markup on wide displays 
See - 
image

**Review by maximprokopchuk — CHANGES_REQUESTED** [review]
Rename files back. No need for camel case in filenames

**Comment by evvis** [comment]
Fixed.

**Review by maximprokopchuk — APPROVED** [review]

**Pull request merged**